src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -231,6 +231,19 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
         })
     }
 
+    fn resolve_path(&self, path_str: &str, ns: Namespace, module_id: DefId) -> Option<Res> {
+        // resolver doesn't know about true and false so we'll have to resolve them
+        // manually as bool
```

Comment:
Can you move this down to the `is_bool_value` line?

Reply:
Yeah I forgot it.